packages/common/src/utils/projectMemberRole.ts

```diff
@@ -0,0 +1,59 @@
+import { OrganizationMemberRole } from '../types/organizationMemberProfile';
+import {
+    GroupRole,
+    InheritedProjectRole,
+    OrganizationRole,
+    ProjectMemberRole,
+    ProjectRole,
+    ProjectRoleOrder,
+} from '../types/projectMemberRole';
+import assertUnreachable from './assertUnreachable';
+
+export const convertOrganizationRoleToProjectRole = (
+    organizationRole: OrganizationMemberRole,
+): ProjectMemberRole | undefined => {
+    switch (organizationRole) {
+        case OrganizationMemberRole.VIEWER:
+            return ProjectMemberRole.VIEWER;
+        case OrganizationMemberRole.INTERACTIVE_VIEWER:
+            return ProjectMemberRole.INTERACTIVE_VIEWER;
+        case OrganizationMemberRole.EDITOR:
+            return ProjectMemberRole.EDITOR;
+        case OrganizationMemberRole.DEVELOPER:
+            return ProjectMemberRole.DEVELOPER;
+        case OrganizationMemberRole.ADMIN:
+            return ProjectMemberRole.ADMIN;
+        case OrganizationMemberRole.MEMBER:
+            return undefined;
```

Comment:
org level member means no permission to the project level!

Reply:
tested that and it works 👍